dev-java/lombok/lombok-1.16.8.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="Project Lombok makes java a spicier language by adding 'handlers' that know how to build and compile simple, boilerplate-free, not-quite-java code."
+HOMEPAGE="https://projectlombok.org/"
+SRC_URI="https://projectlombok.org/downloads/${P}.jar"
+LICENSE="MIT"
+
+SLOT="0"
+KEYWORDS="x86 amd64"
+
+DEPEND=">=virtual/jdk-1.7"
+RDEPEND="${DEPEND}"
```

Comment:
`RDEPEND` should be JRE:
https://wiki.gentoo.org/wiki/Java_Developer_Guide#Dependencies